src/main/java/me/leoko/advancedban/utils/Command.java

```diff
@@ -264,6 +329,8 @@
                     MessageManager.sendMessage(sender, "Check.BanReason", false, "REASON", ban.getReason());
                 }
                 MessageManager.sendMessage(sender, "Check.Warn", false, "COUNT", PunishmentManager.get().getCurrentWarns(uuid) + "");
+
+                MessageManager.sendMessage(sender, "Check.Note", false, "COUNT", PunishmentManager.get().getCurrentNotes(uuid) + "");
```

Comment:
You forgot to add the corresponding message to the message file

Reply:
Will correct